package.json

```diff
@@ -44,7 +44,7 @@
     "@rollup/plugin-commonjs": "^11.0.2",
     "@rollup/plugin-node-resolve": "^7.1.3",
     "@types/sinon": "^9.0.10",
-    "browser-sync": "2.26.10",
+    "browser-sync": "^2.27.9",
```

Comment:
Ask @robertsevernsentient, but I think this upgrade ends up breaking something.

Reply:
@mattstrom -- you're right. IE regression test break. I retract my approval!

Reply:
Updated and tested by running `npm run e2e`

Reply:
@emarkov-evolv -- it will only fail on IE11 - not on a local run. Have you confirmed that?

Previously we have decided this is not worth spending time to resolve the IE11 issue as it's not a production facing change

Reply:
@robertsevernsentient 
Tested by running `npm run e2e`, not sure how I can test for IE11 specifically